<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RatapihaIkkuna</class>
 <widget class="QMainWindow" name="RatapihaIkkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ratapiha</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.RatapihaIkkuna { background: url(:/r/pic/tausta.png) }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>411</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::tab::selected, QTabWidget::pane { background-color: rgba(255, 255, 255, 100); }</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="kaukoTab">
     <attribute name="title">
      <string>Kaukoyhteys</string>
     </attribute>
     <widget class="QPushButton" name="yhdistaNappi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Yhdistä</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/po-aja.png</normaloff>:/r/pic/po-aja.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="katkaiseNappi">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Katkaise yhteys</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/po-seis.png</normaloff>:/r/pic/po-seis.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="ohjausKaukoButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Ohjausnäkymä</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="palvelintab">
     <attribute name="title">
      <string>Paikallinen palvelin</string>
     </attribute>
     <widget class="QPushButton" name="kaynnistaPalvelinNappi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Käynnistä palvelin</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/po-aja.png</normaloff>:/r/pic/po-aja.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pysaytaPalvelinNappi">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Pysäytä palvelin</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/po-seis.png</normaloff>:/r/pic/po-seis.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="rataButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Ratanäkymä</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/varojunaa.png</normaloff>:/r/pic/varojunaa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="ohjausPaikButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Ohjausnäkymä</string>
      </property>
     </widget>
     <widget class="QSlider" name="nopeusSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>361</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Simulaation nopeus</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Muokkaa</string>
     </attribute>
     <widget class="QPushButton" name="muokkaaNakymiaNappi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Muokkaa näkymiä</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/muokkaa.png</normaloff>:/r/pic/muokkaa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="muokkaaRataaNappi">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Muokkaa rataa</string>
      </property>
      <property name="icon">
       <iconset resource="ratapiha-kauko.qrc">
        <normaloff>:/r/pic/muokkaa.png</normaloff>:/r/pic/muokkaa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="asetustab">
     <attribute name="title">
      <string>Asetukset</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Palvelin</string>
           </property>
           <property name="buddy">
            <cstring>palvelinEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="palvelinEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Portti</string>
           </property>
           <property name="buddy">
            <cstring>porttiSpin</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="porttiSpin">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>65000</number>
           </property>
           <property name="value">
            <number>6543</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Käyttäjätunnus</string>
           </property>
           <property name="buddy">
            <cstring>kayttajatunnusLine</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="kayttajatunnusLine"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Salasana</string>
           </property>
           <property name="buddy">
            <cstring>salasanaLine</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="salasanaLine">
           <property name="echoMode">
            <enum>QLineEdit::PasswordEchoOnEdit</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="viestiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none</string>
    </property>
    <property name="text">
     <string>Tervetuloa!</string>
    </property>
   </widget>
   <widget class="QLabel" name="varivalo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ratapiha-kauko.qrc">:/r/pic/ktas_punainen.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="tilaLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Yhteyttä ei ole muodostettu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 125);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_11</zorder>
   <zorder>tabWidget</zorder>
   <zorder>viestiLabel</zorder>
   <zorder>varivalo</zorder>
   <zorder>tilaLabel</zorder>
  </widget>
 </widget>
 <resources>
  <include location="ratapiha-kauko.qrc"/>
 </resources>
 <connections/>
</ui>
